scaleway/resource_iam_group.go

```diff
@@ -139,7 +148,7 @@ func resourceScalewayIamGroupUpdate(ctx context.Context, d *schema.ResourceData,
 		}
 	}
 
-	if d.HasChanges("application_ids", "user_ids") {
+	if !d.Get("external_membership").(bool) && d.HasChanges("application_ids", "user_ids") {
```

Comment:
If we changes from `external_membership = false` to `external_membership = true`, shouldn't the previously set apps and users be deleted?

Reply:
mmmm maybe just clean the state

Reply:
Done as in security group: https://github.com/scaleway/terraform-provider-scaleway/blob/8cdd5f333e5d2f244d7367103f7365f83f2d21cc/scaleway/resource_instance_security_group.go#L292